ext/pcntl/pcntl.stub.php

```diff
@@ -19,8 +19,7 @@ function pcntl_wait(&$status, int $options = 0, &$rusage = []): int {}
 /** @param callable|int $handler */
 function pcntl_signal(int $signo, $handler, bool $restart_syscalls = true): bool {}
 
-/** @return mixed */
-function pcntl_signal_get_handler(int $signo) {}
+function pcntl_signal_get_handler(int $signo): mixed {}
```

Comment:
Maybe `@return callable|int`? It's not completely mixed.

Reply:
Yes, good idea!